lib/manageiq/api/common/rbac/service.rb

```diff
@@ -0,0 +1,66 @@
+require 'rbac-api-client'
+module ManageIQ
+  module API
+    module Common
+      module RBAC
+        class Service
+          def self.call(klass)
+            setup
+            yield init(klass)
+          rescue RBACApiClient::ApiError => err
+            Rails.logger.error("RBACApiClient::ApiError #{err.message} ")
+            raise
+          end
+
+          def self.paginate(obj, method, pagination_options, *method_args)
```

Comment:
This feels like an even higher-level method that just for RBAC Service as everything needs to paginate.  cc @bdunne

Reply:
It feels like something that belongs in an API client

Reply:
Would `method` ever not be `:get`?

Reply:
API client doesn't support pagination, so we add it here.